sass/components/elements.scss

```
@@ -78,26 +78,28 @@ input[type=radio] {
 }
 input[type=checkbox] + label,
 input[type=radio] + label {
+  @extend body;
+  text-transform: none;
   position: relative;
+  padding-left: 12px;
```

```suggestion
  padding-left: 2em;
```
and setting 
```
input[type=checkbox],
input[type=radio] {
  display: none;
}
```
seems to fix it!

---

Nice! I fixed the original code to hide the underlying checkbox by repostioning it. That way users can still tab to that part of the form. I also added a focus style, since I noticed while I was testing that it wasn't getting a visible focus when tabbing.